examples/pod_exec.py

```diff
@@ -117,8 +117,11 @@ def exec_commands(api_instance):
 
 def main():
     config.load_kube_config()
-    c = Configuration()
-    c.assert_hostname = False
+    try:
+        c = Configuration().get_default_copy()
```

Comment:
nit: maybe add a comment that this is the right way of setting the config starting v12.0.0